src/i18n/messages.xlf

```diff
@@ -23,229 +23,410 @@
+        <source>Center Name</source>
         <context-group purpose="location">
-          <context context-type="sourcefile">app\community\community.component.ts</context>
-          <context context-type="linenumber">24</context>
+          <context context-type="sourcefile">app/shared/dialogs/dialogs-view.component.ts</context>
+          <context context-type="linenumber">19</context>
         </context-group>
       </trans-unit>
-      <trans-unit id="eb3240fe1f2eb6fea259628655548655bd59bd68" datatype="html">
-        <source>Last Activities Sync</source>
+      <trans-unit id="9af14d0df3c9c08e3f7f448871fe8c8ce2a5870d" datatype="html">
+        <source>Center Url</source>
         <context-group purpose="location">
-          <context context-type="sourcefile">app\community\community.component.ts</context>
-          <context context-type="linenumber">25</context>
+          <context context-type="sourcefile">app/shared/dialogs/dialogs-view.component.ts</context>
+          <context context-type="linenumber">23</context>
         </context-group>
       </trans-unit>
-      <trans-unit id="030b4423b92167200e39519599f9b863b4f7c62c" datatype="html">
-        <source>Actions</source>
+      <trans-unit id="8fe73a4787b8068b2ba61f54ab7e0f9af2ea1fc9" datatype="html">
+        <source>Version</source>
         <context-group purpose="location">
-          <context context-type="sourcefile">app\community\community.component.ts</context>
-          <context context-type="linenumber">26</context>
+          <context context-type="sourcefile">app/shared/dialogs/dialogs-view.component.ts</context>
+          <context context-type="linenumber">27</context>
         </context-group>
       </trans-unit>
-      <trans-unit id="826b25211922a1b46436589233cb6f1a163d89b7" datatype="html">
-        <source>Delete</source>
+      <trans-unit id="cb33b150a0b9e6b88a9aa5fa79f91888dd06784a" datatype="html">
+        <source>ltr</source>
         <context-group purpose="location">
-          <context context-type="sourcefile">app\community\community.component.ts</context>
-          <context context-type="linenumber">50</context>
+          <context context-type="sourcefile">app/home/home.component.ts</context>
+          <context context-type="linenumber">2</context>
+        </context-group>
+        <context-group purpose="location">
+          <context context-type="sourcefile">app/home/home.component.ts</context>
+          <context context-type="linenumber">3</context>
+        </context-group>
+        <context-group purpose="location">
+          <context context-type="sourcefile">app/login/login.component.ts</context>
+          <context context-type="linenumber">1</context>
         </context-group>
       </trans-unit>
-      <trans-unit id="3f384a474f6473824411ffbd437db55c8a45dad6" datatype="html">
-        <source>Nation List</source>
+      <trans-unit id="46293acefcc2d18110988c5c06447f20e775694d" datatype="html">
```

Comment:
This `id` changed with this pull request, causing translations to not be displayed.  Every language file uses the old `id`, so I will change this file shortly.  It looks like it was possibly changed because we added spaces around ` Nation List `, which we __should not__ do anyway.

@dogi @lmmrssa Is it worth it to check this file further to see if there are other discrepancies, or should we just generate a new one and have Crowdin redo all translations?